README.md

```diff
@@ -14,30 +28,99 @@ To install multiple versions of Node.js, you may find it easier to use a node ve
 * [nvm](https://github.com/creationix/nvm)
 * [n](https://github.com/tj/n)
 
-### Installation
+
+## Installation
+
+Clone this repo and install its dependencies...
 
 ```
 $ npm install
 ```
 
-### Usage
+You'll then need to duplicate the `config.sample.json` file as `config.json` and update its contents...
```

Comment:
I thought about saying rename it, but the problem is that it'll then dirty the git index. `config.json` is gitignored, `config.sample.json` is tracked.